### PR TITLE
Bjorncs/new jdisc metrics

### PR DESCRIPTION
**Pull request by bjorncs**

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java [hunk 1]
```diff
@@ -122,6 +122,8 @@
 
         metrics.add(new Metric("athenz-tenant-cert.expiry.seconds.last", "athenz-tenant-cert.expiry.seconds"));
 
+        metrics.add(new Metric("jdisc.http.request.prematurely_closed.rate"));
```
Comment (gjoranv) [inline config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java]: Please move the `http.status.xxx.rate` metrics here as well, as they are related. I put them in getQrserverMetrics by mistake.

**Review by gjoranv — CHANGES_REQUESTED** [review]

File: config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java (hunk 1 above)
Comment (bjorncs) [reply config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java]: Fixed - PTAL

**Comment by bjorncs** [comment]
@gjoranv PTAL

File: jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestDispatch.java [hunk 2]
```diff
@@ -159,6 +159,7 @@ private ServletRequestReader handleRequest() throws IOException {
         try (ResourceReference ref = References.fromResource(jdiscRequest)) {
             HttpRequestFactory.copyHeaders(jettyRequest, jdiscRequest);
             requestContentChannel = requestHandler.handleRequest(jdiscRequest, servletResponseController.responseHandler);
+            metricReporter.setBindingMatch(jdiscRequest.getBindingMatch());
```
Comment (gjoranv) [inline jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestDispatch.java]: Please add a comment why this must be called after handleRequest

**Review by gjoranv — APPROVED** [review]
Please add comment as mentioned.

**Pull request merged**